libpoolprotocols/PoolManager.cpp

```diff
@@ -507,6 +524,20 @@ void PoolManager::submithrtimer_elapsed(const boost::system::error_code& ec)
     }
 }
 
+void PoolManager::reconnecttimer_elapsed(const boost::system::error_code& ec)
+{
+    if (!ec)
```

Comment:
Use quick exit on error.
```
if (ec)
    return;
```

Reply:
fixed in https://github.com/ethereum-mining/ethminer/pull/1824/commits/eb95ad3ff0c00561f54b881424141c031a677da2